docs/guides/guest-UEFI-Secure-Boot.md

```diff
@@ -4,34 +4,41 @@ How to configure UEFI Secure boot?
 
 The previous Secure Boot settings only included the 2011 Microsoft KEK and db certificates.
-Our defaults now include the 2023 Microsoft certificates, ensuring Secure Boot updates beyond 2026 (which is when the previous 2011 certificates expire).
-These defaults will also be automatically kept up-to-date as XCP-ng is updated.
+Our defaults now include the 2023 Microsoft certificates, ensuring Secure Boot updates beyond 2026 (which is when the previous 2011 certificates expire), while ensuring compatibility with VMs migrated from VMware using XO V2V.
+These defaults will be automatically kept up-to-date as XCP-ng is updated.
 
-Existing installations will not be automatically changed by this update:
-
-* Existing VMs will not be affected unless you use the [Propagate certificates](#propagate-pool-certificates-to-a-vm) feature in Xen Orchestra (which has always had the effect of resetting VM Secure Boot variables to that of the pool).
-* If you followed our previous guides and used `secureboot-certs install` to install the default Secure Boot variables into your pool, these variables will not be changed. However, we recommend clearing the pool variables and receiving the latest ones from XCP-ng updates instead.
+Existing Secure Boot installations will not be automatically updated.
 
 ### Recommended actions
 
-Our defaults will not be applied automatically on existing systems that have been set up for guest Secure Boot.
-Follow the steps below to reset pool Secure Boot configuration to defaults and update existing VMs:
+#### 1. On your XCP-ng pool
 
-1. If you have installed your pool Secure Boot variables using `secureboot-certs install`, use `secureboot-certs clear` to reset them to the XCP-ng-managed defaults.
-  * If you haven't done so, no action is required; your pool now supports guest Secure Boot by default.
+Our defaults will not be applied automatically on existing systems that have been set up for guest Secure Boot.
+If you have installed your pool Secure Boot variables using `secureboot-certs install`, use `secureboot-certs clear` to reset them to the XCP-ng-managed defaults.
+  * If you haven't run `secureboot-certs install` before, no action is required; your pool now supports guest Secure Boot by default.
   * Existing VMs will not be affected by this step.
+  * You can run `secureboot-certs clear` again if unsure.
   * This step is only needed once per pool.
-2. Install the 2023 Microsoft KEK certificate for guest-initiated security updates to the db and dbx variables.
-  * This certificate is **required** on VMs with Secure Boot enabled (Windows **and** Linux) for future Secure Boot updates to succeed.
-  * After clearing the pool variables using the above command, replace VM Secure Boot variables using the [Propagate certificates](#propagate-pool-certificates-to-a-vm) procedure.
+
+#### 2. On new VMs and newly V2V-migrated VMs
+
+No action is required once the pool is reconfigured with the new defaults.
```

Comment:
Mostly true. New VMs created from a user template (created by converting a VM into a template) which has old certificates will be created with the old certificates.